library/core/tests/iter/sources.rs

```diff
@@ -1,5 +1,4 @@
 use super::*;
-use core::iter::*;
```

Comment:
delete it because it resolved at https://github.com/rust-lang/rust/blob/master/library/core/tests/iter/mod.rs#L25

Reply:
Sorry, I don't understand.
Is there an unused import warning if this line it's not removed, or what?

Reply:
In the previous implementation based on sorting to `root_span`, it does give an `unsed import` alert: https://github.com/rust-lang/rust/actions/runs/4450003658/jobs/7814921906#step:26:1943

Reply:
I have since kept this change as it seems to make the code simpler